syntax = "proto3";

package sentinel_ai;

message FilteredEvent {
  string id = 1;
  string title = 2;
  string timestamp = 3;
  string source = 4;
  repeated string categories = 5;
  bool is_relevant = 6;
  // Add other fields as needed that are passed down the stream
}
